downstream/assemblies/platform/assembly-platform-install-scenario.adoc

```diff
@@ -38,6 +38,7 @@ include::platform/ref-standalone-hub-inventory.adoc[leveloffset=+3]
 include::platform/ref-standalone-hub-ext-database-inventory.adoc[leveloffset=+3]
 include::platform/ref-standalone-hub-ext-database-customer-provided.adoc[leveloffset=+3]
 include::platform/ref-ldap-config-on-pah.adoc[leveloffset=+3]
+include::../../../assemblies/hub/assembly-installing-high-availability-hub.adoc[leveloffset=+3]
```

Comment:
This will require a symlink or moving this assembly into the assemblies/platform directory. Note that moving the file to the platform directory could cause broken links that will need to be fixed. @ariordan-redhat or I can help you with this depending on when you're ready.